.github/workflows/staging-deploy.yml

```
@@ -27,6 +27,9 @@ jobs:
       - name: Test formatting with black
         run: |
           black . --check
+      - name: Run pytest
+        run: |
+          pytest
```

```suggestion
          python -m pytest
```

This should fix any import errors.